src/pages/content/voice-and-tone.mdx

```diff
@@ -1,12 +1,98 @@
 ---
 title: Voice and tone
-description: >
-  Lorem ipsum dolor sit amet, consectetur adipiscing
```

Comment:
For the front matter you can use this for the description:

Our product is an extension of our brand. We use visual design to shape what Zendesk looks like, and voice and tone to shape what Zendesk sounds like.